packages/website/src/features/Packages/FunctionOutput.tsx

```diff
@@ -3,6 +3,9 @@ import { Box, Text, Flex } from '@chakra-ui/react';
 import { AbiParameter } from 'abitype';
 import { isArray, isObject } from 'lodash';
 import { InfoOutlineIcon } from '@chakra-ui/icons';
+import { getNumberWithDecimal } from '@/helpers/numbers';
+import { HiCalculator } from 'react-icons/hi';
+import { Tooltip } from 'react-tooltip';
```

Comment:
Unless `react-tooltip` has a really good advantage, could it use https://chakra-ui.com/docs/components/tooltip/usage to avoid adding a new library?

Reply:
Sure! I can give it a try, I was not aware of chakra tooltip 😮 , thank you! @mjlescano